mne/datasets/testing/_testing.py

```diff
@@ -12,6 +12,7 @@
 
 
 has_testing_data = partial(has_dataset, name='testing')
+has_spm_data = partial(has_dataset, name='spm')
```

Comment:
(`_testing` namespace here is for the testing dataset, not all testing-related-things)